packages/web-react/src/components/Skeleton/SkeletonHeading.tsx

```diff
@@ -16,22 +16,26 @@ const defaultProps: Partial<SpiritSkeletonProps> = {
 };
 const SkeletonHeading = <T extends ElementType = 'div', E = void>(props: SpiritSkeletonProps<T, E>): ReactElement => {
   const propsWithDefaults = { ...defaultProps, ...props };
-  const { elementType: ElementTag = defaultProps.elementType as ElementType, lines, ...restProps } = propsWithDefaults;
+  const { elementType: ElementTag = 'div', lines, ...restProps } = propsWithDefaults;
```

Comment:
It would be nice if it would be possible to have it defined only once. Right now, we have twice the `div` in this file. But I understand it is outside of this scope.